source/system-administrators/deployer/processors-guide.rst

```diff
@@ -232,6 +232,39 @@ Processor that updates the processed commits value with the current commit
 
     - processorName: gitUpdateCommitIdProcessor
 
+.. _deployer-script-processor:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+Groovy Script Processor
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A site-specific custom Groovy processor that can process published content.
```

Comment:
I'm not sure `site-specific` is accurate here, both the processor and the scripts can be shared